src/components/form/CheckBox.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { formInput } from '../../utils/react/propTypes'
+
+export const CheckBox = props => (
+    <div className="form-checkbox">
+        <div className="input-container">
+            <input
```

Comment:
Any reason why we are not using `import Checkbox from '@material-ui/core/Checkbox';`?